deploy/charts/cert-manager/values.yaml

```diff
@@ -1,3 +1,7 @@
+# Disables Cert Manager deployment rendering
+certManager:
+  enabled: true
```

Comment:
I'm not sure this should be a new sub-level item.
I wonder what the alternatives are, eg. `enabled` on top level would also be confusing.

Reply:
Suggest alternatives.

Reply:
I tried toding it this way before: https://github.com/cert-manager/cert-manager/pull/4219